client-v2/integration/fixtures/edition.js

```diff
@@ -23,8 +23,7 @@ module.exports = {
           // lastUpdated?: number,
           // updatedBy?: string,
           // updatedEmail?: string,
-          live: [],
-          draft: []
+          articles: []
```

Comment:
Rich, I'm sorry, we just had a chat like 30 minutes ago and decided this should be called items instead of articles. Too fast moving!

Reply:
Haha - no worries, I just guessed this one anyway so it's no big deal 👍